R/frgtracker.R

```diff
@@ -9,16 +9,47 @@
 
 # function3 start
 
+#' Calculate the summary statistics for specified courses including mean, median and quantiles.
+#'
+#' @param course_ids A list including all course IDs for which the summary statistics are
+#' calculated for.
+#'
+#' @return A dataframe containing the summary statistics for specified courses
+#' @export
+#'
+#' @examples
+
+generate_course_statistics <- function(course_ids) {
+  print("NULL")
+}
+
 # function3 end
 
 # function4 start
 
+#' Calculate students' course grades to rank courses in ascending/descending order by a
+#' specified method.
+#'
+#' @param method The method applied to rank the courses, should be "mean", "1st-quantile",
+#' "median", or "3rd-quantile". Defaults to "mean"
```

Comment:
for the method - i would follow a slightly different syntax - I found this [example](https://www.rdocumentation.org/packages/DescTools/versions/0.99.40/topics/PostHocTest) from a function we are using in 554 lab - see the methods part.

Reply:
Thanks for showing me the example, Javaria! I've changed the method part